docs/API/V1/list.md

```diff
@@ -6,7 +6,9 @@ can only list datasets with access `public`.)
 
 The collection of datasets may be filtered using any combination of a number
 of query parameters, including `owner`, `access`, `name` substring, date range,
-and arbitrary metadata filter expressions.
+and arbitrary metadata filter expressions. The selected datasets may be sorted
+by any metadata key value in either ascending or descending order. Multiple
```

Comment:
If the key yields a JSON object, will the Server do something reasonable?  (Should we test that?)

Reply:
I'm not sure whether PostgreSQL JSON sort order is well-defined. It might just work on the serialized substring, for example; and [de]serialization through `json` could change key order. I didn't actually think of that, but I can experiment.